packages/runtime/src/types/transport/RelationshipDTO.ts

```diff
@@ -37,12 +43,18 @@ export interface RelationshipAuditLogDTO extends Array<RelationshipAuditLogEntry
 
 export type RelationshipCreationContentDerivation = RelationshipCreationContentJSON | ArbitraryRelationshipCreationContentJSON;
 
+export interface PeerDeletionInfoDTO {
+    deletionStatus: string;
+    deletionDate: string;
+}
+
```

Comment:
I don't think this is actually required. For the deletionInfo of LocalAttributeDTO we simply use the JSON (which you would need to add).

Reply:
Have a look at this again, please. :)